Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Fix node version due to high potential for incompatibilities.
```

Comment:
Should we put a sample command somewhere (or a script) to run `docker build` with this dockerfile? I'm curious if the user is expected to provide some `context` when running as a part of the command and if they're expected to run it from a particular directory.

Reply:
I added a sample build command, we might have to expand it more later.